tarea.py

```diff
@@ -1,10 +1,12 @@
-nombre=(input ("introduce tu nombre: "))
-año=int(input ("introduce año de nacimiento: "))
-resta=int(input ("introduce año que vas a sumar: "))
-añoa=int(input ("introduce año actual: "))
-edad22=añoa-año
-edadactual=edad22+resta
-edad = resta+año+edad22
```

Comment:
Al importar un módulo desde otro o ejecutarlo como script, el interprete ejecuta el código definido en el.
En muchas ocaciones hay código que no queremos que se ejecute inmediatamente, por ejemplo
estas líneas donde se solicita nombre, edad y año. En este caso solo queremos que se ejecuten solo si el módulo se ejecuta como un script.
En otro escenario tal vez solo nos interesa importar una función que haga el cálculo en un determinado momento bajo otra condición y no inmediatamente.

https://github.com/of-curso-python/curso-python/blob/master/modulos.md